src/Promitor.Agents.Scraper/Dockerfile.linux

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/sdk:7.0.202-alpine3.16 AS build
+FROM mcr.microsoft.com/dotnet/sdk:7.0.302-alpine3.17 AS build
```

Comment:
Upgrading the SDK to 7.0.302 was not required for fixing the vulnerability, but I thought it might be better to change this as well so that the alpine versions are the same in both the build and runtime images.